### PR TITLE
Added skip_if_only_changed to postsubmit builds

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
* Add skip to postsubmit builds

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/prowjobs.md [hunk 1]
```diff
@@ -4,12 +4,13 @@ The Prowjobs in this repo are located under the [jobs folder](../jobs) and are s
...
 ## Job parameters
 
-Each Prowjob is represented as a well-defined set of parameters which are fed as input to a YAML template to generate the final Prowjob YAML. The parameter set is unmarshaled into the following Go struct that corresponds to the standard Prowjob definition fields :
+Each Prowjob is represented as a well-defined set of parameters, which are fed as input to a YAML template to generate the final Prowjob YAML. The parameter set is unmarshalled into the following Go struct that corresponds to the standard Prowjob definition fields :
 ```go
 type JobConfig struct {
 	JobName            string         `json:"jobName,omitempty"`
 	RunIfChanged       string         `json:"runIfChanged,omitempty"`
-	Branches           []string       `json:"branches,omitempty"`
+    SkipIfOnlyChanged  string         `json:"skipIfOnlyChanged,omitempty"`
```
Comment (abhay-krishna) [inline docs/prowjobs.md:12]: Indentation needs to use tags

Comment (abhay-krishna) [reply docs/prowjobs.md:12]: Can we also document this in the table below? I don't think anyone adding a new Prowjob will need this but I feel it should still be in the documentation.

Comment (kschumy) [reply docs/prowjobs.md:12]: yeah for sure

Comment (kschumy) [reply docs/prowjobs.md]: You have no idea the struggle I just went through. But the tabs are fixed

File: docs/prowjobs.md [hunk 2]
```diff
@@ -50,6 +52,11 @@ The configurable parameters are explained in the table below:
 | `volumeMounts` | List of pod volumes to mount into each container in this Prowjob | ✓ | ✓ | ✓ |
 | `volumes` | List of pod volumes that can be mounted by this Prowjob's containers | ✓ | ✓ | ✓ |
 
+Currerntly, `runIfChanged` and `skipIfOnlyChanged` cannot be used at the same time. Upstream Prow
+as put this restriction in place. See the
```
Comment (abhay-krishna) [inline docs/prowjobs.md:56]:
```suggestion
has put this restriction in place. See the 
```

**Pull request merged**